.gitignore

```
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
```

Haría falta ignorar el directorio `coverage` para no subirlo por error, este directorio se genera al lanzar el comando que has definido en el fichero `package.json` `npm run coverage`
```suggestion
coverage
*.local
```

---

Ya lo tienes